tests/helpers/test_entity_component.py

```diff
@@ -441,28 +457,27 @@ def async_loop_exception_handler(_, _2) -> None:
     """Test extract all with None and *."""
     component = EntityComponent(_LOGGER, DOMAIN, hass)
     await component.async_add_entities(
-        [MockEntity(name="test_1"), MockEntity(name="test_2")]
-    )
+        [MockEntity(name="test_1"),
+         MockEntity(name="test_2")])
 
     call = ha.ServiceCall("test", "service")
 
     assert [] == sorted(
-        ent.entity_id for ent in await component.async_extract_from_service(call)
-    )
+        ent.entity_id
+        for ent in await component.async_extract_from_service(call))
 
 
 async def test_extract_all_use_match_all(hass, caplog):
     """Test extract all with None and *."""
     component = EntityComponent(_LOGGER, DOMAIN, hass)
     await component.async_add_entities(
-        [MockEntity(name="test_1"), MockEntity(name="test_2")]
-    )
+        [MockEntity(name="test_1"),
+         MockEntity(name="test_2")])
 
     call = ha.ServiceCall("test", "service", {"entity_id": "all"})
 
     assert ["test_domain.test_1", "test_domain.test_2"] == sorted(
-        ent.entity_id for ent in await component.async_extract_from_service(call)
-    )
-    assert (
-        "Not passing an entity ID to a service to target all entities is " "deprecated"
-    ) not in caplog.text
+        ent.entity_id
+        for ent in await component.async_extract_from_service(call))
+    assert ("Not passing an entity ID to a service to target all entities is "
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4603109)